src/AddOn.Episerver.Settings/Core/DefaultSettingsResolver.cs

```diff
@@ -0,0 +1,38 @@
+﻿using EPiServer;
+using EPiServer.Core;
+
+namespace AddOn.Episerver.Settings.Core;
+
+public class DefaultSettingsResolver : ISettingsResolver
```

Comment:
Please add XML-comments to all public APIs/members.

Reply:
Done